csrc/scheduler/pointwise_utils.cpp

```diff
@@ -50,6 +50,22 @@ bool DomainMap::areAllInputIdsMappedTo(TensorView* input_tv, TensorView* tv)
   eraseIfInputMappedThroughRFactorDomain(
       in_concrete_ids, tv->getMaybeRFactorDomain());
 
+  // Erase input concrete IDs mapped to any reduction domains
```

Comment:
I think we can just ignore any reduction domains as they should not show up in the fusion after reduction-to-full translations.

Reply:
Couldn't you simply ignore any input tensor view that doesn't have any `uses()`?

Reply:
This function needs to check that all input IDs are represented by the reference tensor. Any input with empty `uses()` would violate that since its IDs would not map to any others. You're right that this is safe, and is already in place at the one call-site: https://github.com/NVIDIA/Fuser/blob/d0214274982a8d40c2bfeb5047cfc5bd130379ad/csrc/scheduler/pointwise_utils.cpp#L124-L127. Note that more generally, we could use `InputsOf(fusion->outputs())` so that empty uses propagate back to give us more ignored inputs.

The check here is actually skipping any input IDs that map to internal reductions (not represented in outputs. That seemed like a good idea at the time but now I realize that's a bad idea, since we do still need those IDs represented in the reference tensor.